packages/vite/src/client/overlay.ts

```diff
@@ -124,7 +140,7 @@ code {
     <pre class="frame" part="frame"></pre>
     <pre class="stack" part="stack"></pre>
     <div class="tip" part="tip">
-      Click outside or fix the code to dismiss.<br>
+      Click outside, press <span class="kbd">Esc</span> key, or fix the code to dismiss.<br>
```

Comment:
I mean we should use the [kbd tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) here, like `<kbd>Esc</kbd>`. The rest of the changes looks great.

Reply:
Oops, my bad. I didn't know that was a thing. Thank you for explaining it :D

Changed it right away.